amplify/backend/function/stockerlambda/ts/yahoo.ts

```diff
@@ -69,6 +67,30 @@ export async function getPrice(ticker: string, when?: number): Promise<Price> {
   };
 }
 
+function formatTicker(ticker: string): { ticker: string, transform: Function } {
+  if (ticker === 'SGDCAD=X') {
+    return {
+      ticker: 'SGDCAX=X',
+      transform: (n: number) => n,
+    }
+  }
+
+  if (
+    /[EUR|USD|SGD]=X$/.test(ticker)
+    && !(/^[EUR|USD|SGD]/.test(ticker))
```

Comment:
If ticker ends in EUR USD or SGD but doesn't start with any of those, then reverse the ticker so we get reliable rates from yahoo